csharp/Program.cs

```diff
@@ -4,9 +4,58 @@ namespace csharp
 {
     class Program
     {
+        
+        static String numbercard;
+        static int auxnumbercard;
+
         static void Main(string[] args)
         {
-            Console.WriteLine("Hello C#!");
+            while (true)
+            {
+                Console.WriteLine("Write your credit card number");
+                numbercard = Console.ReadLine();
+                getIssuer(numbercard);
+            }
+        }
```

Comment:
preferably, keep one blank line between method definitions. Those help to notice where a function ends and another one begins.